index.ts

```diff
@@ -57,20 +78,22 @@ class Analytics {
       logLevel = bunyan.FATAL,
     }: {
       enable?: boolean;
-      timeout?: number;
+      timeout?: number | string;
```

Comment:
is there some reason we want to support a string here if it needs to be a number in the fetch request?

Reply:
I think this was a bad merge conflict. We should switch to numbers only and drop `ms`. Callers can use ms directly if they want.

Reply:
Actually, we still support strings. Will keep this for now and remove the ms behavior in a dedicated PR.

Reply:
sounds good. I think we should add something like `timeout = Number(timeout)` to make sure it's a `number` internally. (maybe in a future PR)

Reply:
If all of our call sites use numbers, let's drop support for strings in a breaking change. Coercing strings to numbers with Number instead of ms would be a breaking change anyway.